lib/linter.js

```diff
@@ -996,13 +1006,15 @@ module.exports = class Linter {
             throw err;
         }
 
-        return applyDisableDirectives({
+        lintingProblems = applyDisableDirectives({
```

Comment:
Nitpick: Could this just be `return applyDisableDirectives({ // ...` again since `lintingProblems` is no longer being modified afterwards?